scripts/script.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-2-Clause
```

Comment:
please give this file a more descriptive name

Reply:
Made all script names more descriptive